go-server-server/go/auth.go

```diff
@@ -0,0 +1,24 @@
+package mseeserver
+
+import (
+    "log"
+    "net/http"
+)
+
+func CommonNameMatch(r *http.Request) bool {
```

Comment:
Do you want to keep this function name or plans to change in later commits?

Reply:
Being inside the auth file, I prefer this name because its exactly what the function is doing.